core/tests/test_utils.py

```diff
@@ -53,6 +55,9 @@
 
 
 def empty_environ():
+    """Set up empty environment variables for Tests
+        (used in AppEngineTestBase).
```

Comment:
Move what you can to the previous line, as long as it doesn't go over 80 chars.

Also deindent so that left edge of this line matches left edge of previous line.

Reply:
corrected throughout file

Reply:
corrected throughout file